bucket/clash-for-windows.json

```diff
@@ -0,0 +1,29 @@
+{
+    "homepage": "https://github.com/Fndroid/clash_for_windows_pkg",
```

Comment:
This homepage is completely useless.

There is no information, nothing.

![image](https://user-images.githubusercontent.com/13260377/98087937-1d880500-1e81-11eb-8274-7a275f419919.png)

Reply:
It's the only release place of this app, maybe it can be called a homepage? And there is a doc page using gitbook, that's the url in the description before. [https://docs.cfw.lbyczf.com/](https://docs.cfw.lbyczf.com/). Maybe I should put this in the homepage?

Reply:
I would not add this into official bucket.

There is no real homepage. Other "homepages" are in Chinese. This is unacceptable, as majority of users cannot properly use this.